marimbabot_audio/README.md

```diff
@@ -0,0 +1,133 @@
+# TAMS Master Project 2022/2023
+
+This repository contains the codebase for a [Marimba](https://en.wikipedia.org/wiki/Marimba) playing robot developed during the TAMS Master Project 2022/2023. 
```

Comment:
I think you can just link the main README.md instead of copying this part. This should make it easier to keep everything up to date.

Reply:
done, only use the main README.md for now